packages/plugin-default-event-tracking-advanced-browser/src/default-event-tracking-advanced-plugin.ts

```diff
@@ -259,20 +257,27 @@ export const defaultEventTrackingAdvancedPlugin = (options: Options = {}): Brows
           });
         });
       });
-      if (document.body) {
-        observer.observe(document.body, {
+    }
+    if (document.body) {
+      const allElements = querySelectUniqueElements(document.body, cssSelectorAllowlist);
+      allElements.forEach(addListener);
+      /* istanbul ignore next */
+      observer?.observe(document.body, {
+        subtree: true,
+        childList: true,
+      });
```

Comment:
Nit: put these block into a function so that it can be reused in the else condition.

Reply:
Updated, thanks!